### DISCUSSION TITLE
Surface 3 - How far can I update to

### DISCUSSION THREAD

**Discussion by CentbairD** [sebanc/brunch#1885] [Q&A]
I have a slight question. I have a Surface 3 (non-pro) tablet thing. The one that has the screen and only that. After a few headache tries with the current builds and brunch. I saw someone back three years ago with an older build. That uses 'Rammus 83' and the last version of Brunch R83 that came out. Hopefully that makes sense.  I am wondering how far I can go in updates or is there a certain version that stops at the Surface 3? 
CPU is Intel Atom x7 Z8700

**Comment by micindev** [comment]
brunch_r109_stable_20230201 and chromeos_15236.80.0_rammus_recovery_stable-channel_mp-v2.bin works fine so far, some android apps may not work, out of the box no netflix in play store

*edit
after some testing, well, If you have 2gb ram surface, I wouldn't bother, 4gb s3 have only around 300mb of ram free after boot, CPU utilization is around 30% just by doin nothing, there is ZERO performance benefit over windows 11

**Comment by Warrentheo** [comment]
I have a S3 Pro with an i7-4650U 2x2 core with 8gb, I upgraded Brunch recently and found out that the 4th gen Haswells were dropped from the supported list :-/
Until then it has been a great triple boot system, but Windows is finally painful and Brunch no longer boots, leaving just Linux 🥲
